Formula/etcd.rb

```diff
@@ -16,9 +16,9 @@ class Etcd < Formula
 
   depends_on "go" => :build
 
-  # remove in the next release
+  # See removed after https://github.com/etcd-io/etcd/pull/11937
```

Comment:
I think this comment was a little misleading as there's no indication that the pull request has been looked at or will be merged.